Gemfile

```diff
@@ -0,0 +1,48 @@
+source "https://rubygems.org"
```

Comment:
What is the goal of including the `Gemfile` here? I think we need to be careful since github actions is quite finicky on what it accepts/uses. I'm not sure if this will work for the actual website.

Reply:
Locking the versions means the builds are more reproducable, and it also made it easier to use a preview service (Vercel in this case, see https://duckdb-web.vercel.app).

Happy to remove however

Reply:
Ah, that seems fine to me.